### PR TITLE
MINOR: Fix sasl.jaas.config doc string

### PR DESCRIPTION
**Pull request by omkreddy**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by omkreddy** [comment]
@rajinisivaram Pls take a look at this minor PR.

**Comment by omkreddy** [comment]
retest this please

File: clients/src/main/java/org/apache/kafka/common/config/SaslConfigs.java [hunk 1]
```diff
@@ -49,7 +49,7 @@
     public static final String SASL_JAAS_CONFIG = "sasl.jaas.config";
     public static final String SASL_JAAS_CONFIG_DOC = "JAAS login context parameters for SASL connections in the format used by JAAS configuration files. "
         + "JAAS configuration file format is described <a href=\"http://docs.oracle.com/javase/8/docs/technotes/guides/security/jgss/tutorials/LoginConfigFile.html\">here</a>. "
-        + "The format for the value is: '<loginModuleClass> <controlFlag> (<optionName>=<optionValue>)*;'. For brokers, "
+        + "The format for the value is: '&lt;loginModuleClass&gt; &lt;controlFlag&gt; (&lt;optionName&gt;=&lt;optionValue&gt;)*;'. For brokers, "
```
Comment (rajinisivaram) [inline clients/src/main/java/org/apache/kafka/common/config/SaslConfigs.java]: @omkreddy Thanks for the PR. Perhaps we should just use `<code>loginModuleClass controlFlag (optionName=optionValue)*;</code>`, making it easy to read from code as well as doc? What do you think?

**Comment by omkreddy** [comment]
@rajinisivaram Thanks for the review. makes sense. updated the PR.

**Review by rajinisivaram — APPROVED** [review]
@omkreddy Thanks for the update, LGTM.

**Pull request merged**